quickstart/dotnet-core/README.md

```diff
@@ -18,26 +18,46 @@ This sample shows you how to incorporate Azure Cache for Redis into a C# .NET Co
 - Azure subscription - [create one for free](https://azure.microsoft.com/free/)
 - Azure Cache for Redis cache - [create one](https://docs.microsoft.com/azure/azure-cache-for-redis/quickstart-create-redis)
 - [.NET Core SDK](https://dotnet.microsoft.com/download)
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
 
 ## Run the sample
 
 [Download the sample code to your development PC.](/README.md#get-the-samples)
 
-In your command window, change directories to the folder containing this sample.
+### 1. Set up local credential for using Entra ID
+This sample uses Microsoft Entra ID for connecting to an Azure Cache for Redis instance.
+The following line of code in *ContosoTeamStats/RedisConnection.cs* obtains the default credential from your local machine or an Azure resource as the identity for authentication and authorization.
 
-Execute the following command to restore the packages:
+```csharp
+var configurationOptions = await ConfigurationOptions.Parse($"{_redisHostName}:6380").ConfigureForAzureWithTokenCredentialAsync(new DefaultAzureCredential());
+```
+
+One of the common way for signing into to your Azure account is to use the Azure CLI. Bring up the Command Prompt. Run
```

Comment:
`ways`

Reply:
Why not let users us `azd` to be consistent with other samples

Reply:
fixed.